app/actions/labRecords.js

```diff
@@ -113,6 +127,13 @@ export const fetchLabRecord = labRecordId => async (dispatch, getState) => {
   return labRecord;
 };
 
+export const JSONFile = (content, filenme) => {
+  const blob = new Blob([JSON.stringify(content)], {
+    type: 'application/json'
+  });
+  return new File([blob], filenme);
+};
```

Comment:
Food for thought, not a request for change - could this be another `utils` method?

Reply:
Yep. I think the same but have been lazy about it. I didn't find the proper `utils` file so I had to create a new one. But let's do it!

Reply:
What do you think about renaming `isDate.js` file to a new more general name such as `utils.js` and keep both methods there?

Reply:
I think from the folder structure that you're meant to make a new file within the `utils` folder for every 'topic' you're addressing with your subset of methods.

From my experience in projects, you often either have one `utils` file (if you have few methods) or a `utils` folder with several, differently-named files for each subset of logics you're covering!

Reply:
Good to know, thanks!